### ISSUE TITLE
Doesn't handle symbolic links

### ISSUE THREAD

**Issue by philn** [flattool/ignition#6]
Before finding this nice app I was symlinking .desktop files to `~/.config/autostart/`. Ignition doesn't list those entries.

**Comment by heliguy4599** [comment]
Yeah this is because of how the Flatpak sandbox works. Ignition cannot reach files that aren't within its sandbox. The only way around this is to expand Ignition's reach with something like Flatseal.
